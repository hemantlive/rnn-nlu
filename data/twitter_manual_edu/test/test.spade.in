<s> It was Apple Store in ManchesterArndale, I had silver phone and was given a rose gold. Now in USA and wasn't sure how I can get it swapped? </s>
<s> @AppleSupport why is voice searching not working on my Apple TV? It just hangs then crashes </s>
<s> All my brother and sister wanted was a street wise 2 and an avocado twista. what they got was unacceptable and unhygienic. </s>
<s> @kfc fast food at KFC Glastonbury? What a joke , waited 10 minutes after ordering to be told at the window it's a 20 min wait </s>
<s> @Citakki Good to know. Do you see any error messages in Nike+ Connect when trying to sync? </s>
<s> @markirv11 Sorry to hear that. Give these steps a shot and let us know how it goes: http://gonike.me/6016dxJ6 . Any luck? </s>
<s> Don’t sync just yet, we’re taking some time to work on Nike+. You down to hit the road?  </s>
<s> Let's look into what's going on with this. Could you explain the steps you're taking to copy and which app you are using? </s>
