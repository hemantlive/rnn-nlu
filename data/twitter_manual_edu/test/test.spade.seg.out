It was Apple Store in ManchesterArndale , 
I had silver phone and was given a rose gold. 
Now in USA 
and was n't sure how I can get it swapped ? <S>
@ AppleSupport 
why is voice searching not working on my Apple TV ? 
It just hangs then crashes <S>
All my brother and sister wanted was a street wise 2 and an avocado twista. 
what they got was unacceptable and unhygienic . <S>
@ kfc 
fast food at KFC Glastonbury ? 
What a joke ,
waited 10 minutes after ordering to be told at the window it 's a 20 min wait <S>
@ Citakki 
Good to know. 
Do you see any error messages in Nike+ Connect when trying to sync ? <S>
@ markirv11 
Sorry to hear that. 
Give these steps a shot and let us know how it goes : http : //gonike.me/6016dxJ6 . <S>
Don’t sync just yet , 
we’re taking some time to work on Nike+. 
You down to hit the road ? <S>
Let 's look into what 's going on with this. 
Could you explain the steps you 're taking to copy and which app you are using ? <S>
